packages/frontend/src/components/claims/chains/styles.module.css

```diff
@@ -0,0 +1,19 @@
+.root {
+    @apply w-72 flex flex-col gap-2 bg-white rounded-xl p-4;
+}
+
+.header {
+    @apply mb-4;
+}
+
+.row {
+    @apply flex p-3 gap-3 rounded-lg hover:bg-gray-100 cursor-pointer;
```

Comment:
The usual duration/transitions classes where we change properties on hover.

Reply:
Addressed in fc2090d0f245c01946f822083b8adbd743cd8f78